content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -350,8 +350,8 @@ StatefulSet will then begin to recreate the Pods using the reverted template.
 The optional `.spec.persistentVolumeClaimRetentionPolicy` field controls if
 and how PVCs are deleted during the lifecycle of a StatefulSet. You must enable the
 `StatefulSetAutoDeletePVC` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-to use this field. Once enabled, there are two policies you can configure for each
-StatefulSet:
+on the API server and the controller manager to use this field. 
```

Comment:
Would it be safer to say to enable the feature gate on all k8s components? And in general isn't it best practice to have feature gates consistent across all components anyway? (This last question directed more at website approvers who might know standard practice better).

Reply:
As @mengjiao-liu pointed out, other features like https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-tracking-with-finalizers specify the components to set the feature gate on. I'm not sure I agree with that, but I'll defer to standard practice.

/lgtm

Reply:
Seek more advice
 /cc @sftim  @tengqm